images/ceph/Makefile

```
@@ -18,9 +18,9 @@ include ../image.mk
 # Image Build Options
 
 ifeq ($(GOARCH),amd64)
-CEPH_VERSION ?= v16.2.7-20220216
+CEPH_VERSION = v17.2.0-20220420
```

I believe we want to keep the `?=` syntax so custom builds could override this if desired
```suggestion
CEPH_VERSION ?= v17.2.0-20220420
```